.licensee.json

```diff
@@ -2,13 +2,18 @@
   "licenses": {
     "spdx": [
       "CC-BY-3.0",
-      "CC0-1.0"
+      "CC0-1.0",
+      "Unlicense"
     ],
     "osi": true
   },
   "packages": {
     "deep-is": "=0.1.3",
     "docopt": "=0.6.2",
     "prelude-ls": "=1.1.2"
-  }
+  },
+  "ignore": [
+    {"prefix": "querystring"},
+    {"prefix": "json-schema"}
+  ]
```

Comment:
Please move these to packages exceptions instead of ignore.  We don't want to ignore the packages if they change the licence in a later version.  Licences were manually reviewed for current version of these packages.